src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -1,34 +1,96 @@
 import React from 'react';
 import styled from 'styled-components';
-import { shape, string } from 'prop-types';
+import { shape, string, node } from 'prop-types';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/dist/spacings';
 import { GridItemConstrainedMedium } from '#lib/styledGrid';
 
+const KEYLINE_WIDTH = '0.125rem';
```

Comment:
What does 'KEYLINE' refer to is that a design term?

Reply:
Yes, it refers to the blue horizontal line coming out of the show more button